delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -0,0 +1,370 @@
+import six
+
+from oslo_log import log as logging
+from delfin import exception
+from delfin.common import constants, alert_util
+from delfin.drivers.utils.ssh_client import SSHPool
+from delfin.drivers.utils.tools import Tools
+
+try:
+    import xml.etree.cElementTree as ET
+except ImportError:
+    import xml.etree.cElementTree as ET
+
+LOG = logging.getLogger(__name__)
+
+
+class SSHHandler():
+
+    SECONDS_TO_MS = 1000
+
+    DISK_PHYSICAL_TYPE = {
+        'fc': constants.DiskPhysicalType.FC,
+        'SAS': constants.DiskPhysicalType.SAS
+    }
+
+    def __init__(self, **kwargs):
+        self.ssh_pool = SSHPool(**kwargs)
+
+    def login(self):
+        try:
+            result = self.ssh_pool.do_exec('show pools')
+            if 'is not a recognized command' in result:
+                raise exception.InvalidIpOrPort()
+        except Exception as e:
+            LOG.error("Failed to login msa storwize_svc %s" %
```

Comment:
Why there is `storwize_svc` in this log?

Reply:
Forget to delete the error log. It will be updated next time